### PR TITLE
More submit docs

### PR DESCRIPTION
**Pull request by sbreiff**
In this PR:
- support email address changed in md docs
- readthedocs link added for SubmitCGAP documentation
- a little extra detail added in md files concerning CLI submission of cases and family history

### CODE REVIEW

File: docs/public/help/accessioning_3.md [hunk 1]
```diff
@@ -1,38 +1,58 @@
...
  to coordinate file transfer.
 
 #### Option 2: Submission with the python package
 
-We have developed a python package for submission of 
-accessioning spreadsheets to the CGAP portal, called 
-SubmitCGAP. The current version is still in beta. It can be
+We have developed a python package for submission of
+ accessioning spreadsheets to the CGAP portal, called
+ SubmitCGAP. The current version is still in beta. It can be
```
Comment (drio18) [inline docs/public/help/accessioning_3.md:34]: Is SubmitCGAP still considered a beta version at this point?

File: docs/public/help/family_history_3.md [hunk 2]
```diff
@@ -1,31 +1,49 @@
...
+ recommend doing this inside a virtual environment running
+ python 3.6. Further instructions for submission with this
+ method can be found in the [SubmitCGAP documentation](https://submitcgap.readthedocs.io/en/latest/getting_started.html#family-history).
+
+Briefly, the command is the same as for new case/accessioning CLI submission, but
+ with the added `--ingestion_type` option:
+
+```
+
+submit-metadata-bundle <pedigree.xlsx> -s <CGAP_server_url> --ingestion_type family_history -v
```
Comment (drio18) [inline docs/public/help/family_history_3.md:38]: Consider mentioning one can use `-t` in place of `--ingestion_type` here as all other args are given in short format.

**Review by drio18 — APPROVED** [review]
Looks good. As before, the pages will have to be manually re-indexed to show the changes in the markdown files.

File: docs/public/help/accessioning_3.md (hunk 1 above)
Comment (sbreiff) [reply docs/public/help/accessioning_3.md:34]: I asked Kent for his opinion:
"i think it’s as beta as the rest of the system. (i personally consider the rest of the system to be in beta. what we are doing with our monthly dev meetings is what in business would be a beta. a beta is a real release but to a limited audience.)"

certainly i think there are things we could still improve and user feedback we haven't implemented yet so I think its alright to keep this in.

**Pull request merged**